### PR TITLE
Version Bumps

### PR DESCRIPTION
**Pull request by faddat**
- Version Bumps, done nmore cleanly.
- change versions of golang and replace cosmos-sdk
- update go version & copyright date in Readme.md
- Requested modifications
- These version bumps will improve performance and stability and (in other chains) have proven to be non-state-breaking.

## Summary of changes

<Describe summary of major changes here>

## Report of required housekeeping

- [ ] Github issue OR spec proposal link
- [ ] Wrote tests
- [ ] Updated API documentation (client/lcd/swagger-ui/swagger.yaml)
- [ ] Added a relevant changelog entry: clog add [section] [stanza] [message]

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+go 1.17
 
 module github.com/terra-money/core
 
 require (
 	github.com/CosmWasm/wasmvm v0.16.1
-	github.com/cosmos/cosmos-sdk v0.44.2
-	github.com/cosmos/ibc-go v1.1.0
+	github.com/cosmos/cosmos-sdk v0.44.5
+	github.com/cosmos/ibc-go v1.2.4
```
Comment (YunSuk-Yeo) [inline go.mod:8]: We made new branch for next update `release/v0.6.x` and it will be good to update ibc-go version to that branch.

**Review by YunSuk-Yeo — CHANGES_REQUESTED** [review]
there are some updates already applied

File: go.mod (hunk 1 above)
Comment (faddat) [reply go.mod:8]: great idea!

I shouldn't have changed the version number there, anyhow.  

If we want to target wasm 1.0, we should also plan to update to sdk v0.45.0

**Comment by faddat** [comment]
no longer relevant

**Pull request closed without merging**